commit 24c314aae74dd924c670b5249ec1c8a8235d504c
Author: Trent Mick <trentm@gmail.com>
Date:   2019-07-17T10:55:04-07:00 (3 months ago)
    
    TRITON-1818 explicitly specify PEM key format for keys generated by ssh-keygen
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/lib/rbac.js b/lib/rbac.js
index 4c595e3..14c0f41 100644
--- a/lib/rbac.js
+++ b/lib/rbac.js
@@ -800,7 +800,7 @@ function executeRbacUpdatePlan(ctx, cb) {
                     },
                     function generateSshKey(ctx2, next2) {
                         var cmd = format(
-                            'ssh-keygen -t rsa -C "%s" -f %s -b 4096',
+                            'ssh-keygen -t rsa -m PEM -C "%s" -f %s -b 4096',
                             ctx2.keyName, ctx2.keyPath);
                         console.log('    Generate 4096 bit RSA key: %s[.pub]',
                             ctx2.keyPath);
